### ISSUE TITLE
Add output to indicate type-checking was successful on `stdout`

### ISSUE THREAD

**Issue by MatthewDaggitt** [vehicle-lang/vehicle#219]

**Comment by wenkokke** [comment]
If this is regarding the golden tests, we could add a flag to the test files of type `boolean | number` which gets compared to the exit code? (`true` meaning `0`, `false` meaning non-zero, `number` being compared exactly.)

**Comment by MatthewDaggitt** [comment]
No, just in general. If you run Vehicle in type-check mode and it passes it doesn't produce any output. Would be nice to print type-checking successful to let the user know that it hasn't silently errored.
